features/roomlist/impl/src/main/kotlin/io/element/android/features/roomlist/impl/RoomListView.kt

```diff
@@ -204,16 +205,21 @@ private fun RoomListContent(
     Scaffold(
         modifier = modifier.nestedScroll(scrollBehavior.nestedScrollConnection),
         topBar = {
-            RoomListTopBar(
-                matrixUser = state.matrixUser,
-                showAvatarIndicator = state.showAvatarIndicator,
-                areSearchResultsDisplayed = state.searchState.isSearchActive,
-                onToggleSearch = { state.eventSink(RoomListEvents.ToggleSearchResults) },
-                onMenuActionClicked = onMenuActionClicked,
-                onOpenSettings = onOpenSettings,
-                scrollBehavior = scrollBehavior,
-                displayMenuItems = !state.displayMigrationStatus,
-            )
+            Column {
+                RoomListTopBar(
+                    matrixUser = state.matrixUser,
+                    showAvatarIndicator = state.showAvatarIndicator,
+                    areSearchResultsDisplayed = state.searchState.isSearchActive,
+                    onToggleSearch = { state.eventSink(RoomListEvents.ToggleSearchResults) },
+                    onMenuActionClicked = onMenuActionClicked,
+                    onOpenSettings = onOpenSettings,
+                    scrollBehavior = scrollBehavior,
+                    displayMenuItems = !state.displayMigrationStatus,
+                )
+                if (state.displayFilters) {
+                    RoomListFiltersView(state = state.filtersState)
+                }
+            }
```

Comment:
The filter will be displayed above the verify session banner, right? This is a bit strange, no?

Reply:
Yes, I think it's OK. But can review that with the design team.

Reply:
Putting a screenshot here, for the record:

<img width="430" alt="image" src="https://github.com/element-hq/element-x-android/assets/3940906/e01a5ff1-da79-4045-958c-50ed9624785d">

There is less and less space for the rooms :)

CC @amshakal